pkg/serverless/trigger/events.go

```diff
@@ -294,3 +295,48 @@ func eventRecordsKeyEquals(event map[string]any, key string, val string) bool {
 	}
 	return false
 }
+
+func (et AWSEventType) String() string {
```

Comment:
This is only there to make the debug log of detection easier to read/reason about.